plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DockerConfigMetaData.java

```diff
@@ -61,16 +61,25 @@ public DockerConfigMetaData(IDockerImageInfo info) {
 	
 	@Override
 	public Set<String> exposedPorts() {
+		if (info == null ) {
+			return null;
```

Comment:
not a good idea because we want to signal via null (or exception) that no metadata is present.